gke/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: clip-search
+  name: clip-search
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: clip-search
+  template:
+    metadata:
+      labels:
+        app: clip-search
+    spec:
+      containers:
+      # image registry URI should be replaced once GitHub Action is set up
```

Comment:
Why so?

Reply:
it is absolutely fine with leaving this as is static. However, registry URI can be changed if we choose different tag names, or other hosting services, etc. So it is best practice to make this configurable I guess.

Reply:
Got it.